datadog/resource_datadog_integration_pagerduty_service.go

```diff
@@ -0,0 +1,114 @@
+package datadog
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/zorkian/go-datadog-api"
+)
+
+func resourceDatadogIntegrationPagerdutyService() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDatadogIntegrationPagerdutyServiceCreate,
+		Read:   resourceDatadogIntegrationPagerdutyServiceRead,
+		Exists: resourceDatadogIntegrationPagerdutyServiceExists,
+		Update: resourceDatadogIntegrationPagerdutyServiceUpdate,
+		Delete: resourceDatadogIntegrationPagerdutyServiceDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceDatadogIntegrationPagerdutyServiceImport,
```

Comment:
As this importer function does nothing special, it could be replaced by the generic importer:
```suggestion
			State: schema.ImportStatePassthrough,
```